lib/help_scout/api.rb

```diff
@@ -49,12 +49,11 @@ def new_connection
     end
 
     def send_request(action, path, params)
-      connection = new_connection
-      response = connection.send(action, path, params.compact)
+      response = new_connection.send(action, path, params.compact)
 
       if response.status == 401
-        HelpScout::API::AccessToken.refresh!
-        response = connection.send(action, path, params.compact)
```

Comment:
@jcypret @prsimp this is actually the root of the issue in prod, we refresh the token, but don't build a new connection. The connection we previously built has the old token 🤦‍♂️I fixed it in this commit, but ran out of time to add the spec coverage I'd like to add. We can either add that in a later PR (like when we add caching) or maybe @prsimp has time?

Reply:
Ah! Good find. That makes sense we'd want to grab a new connection.